### ISSUE TITLE
Is there any plan to add the training code?

### ISSUE THREAD

**Issue by lixiangweiabc** [zju3dv/4K4D#13]
Thanks for sharing！I was so interested in your job that I wanted to try it on my own data.

**Comment by dendenxu** [comment]
Certainly! I'm working on adding examples and documentations for the training process right now but same as before I personally still couldn't give a guarantee as to the exact date. Stay tuned!

**Comment by dendenxu** [comment]
Hi @lixiangweiabc, I recently released the training code and documentation for training on custom datasets, check it out in our [readme](https://github.com/zju3dv/4K4D/blob/main/readme.md#Training).
Feel free to drop in a comment if you encounter any issues and I'll be happy to help.
